examples/voice_send.cr

```diff
@@ -35,6 +37,17 @@ client.on_message_create do |payload|
     client.create_message(payload.channel_id, "Connecting...")
     connect_channel_id = payload.channel_id
     client.voice_state_update(ids[0].to_u64, ids[1].to_u64, false, false)
+  elsif payload.content.starts_with? "!vs"
+    # Used as:
+    # !vs
+
+    reply = begin
+      vs = cache.resolve_voice_state(payload.guild_id.not_nil!, payload.author.id)
+      "You are connected to channel #{vs.channel_id} at guild #{vs.guild_id}"
+    rescue
+      "No voice state"
+    end
+    client.create_message(payload.channel_id, reply)
```

Comment:
I'd rather we didn't throw an exception and rescue it like this.
Prefer if we just make `resolve_voice_state` nilable and then do something like so:
```cr
reply = if vs = cache.resolve_voice_state(...)
              "You are connected to channel ...."
            else
              "No voice state"
            end

client.create_message(payload.channel_id, reply)

Reply:
As far as I remember, all of `resolve_*` methods in the lib do not return `nil`

Reply:
meh. That's fair, alright.
I may change that in a future PR.